Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Gorian Empire

## Changelog:
- The ogres can restore the Gorian Empire. It works similar to the Roman Empire: imperator's children are born in the purple, has a special elective succession and minor titles.
- Ogre culture has a good opinion of two-headed ogres and prefers to vote for them.

## Tasks:
<details>
<summary>Click to expand</summary>

- [x] Add bloodline
- [x] Write notification text
- [x] Let ogres use roman titles
- [x] Let Gorian Empire use Imperial government and succession law
- [x] Rename capital holding in capital and make sure capital is castle
- [x] Imperor can pick Arcane religion
- [x] Adapt Born in Purple
- [x] Change government picture
- [x] Add decision icon
- [x] Check roman scripted triggers

</details>


# How to test:
Start as the Gordunni Clan in 605's Feralas.

`usurp e_eldrethalas` to get an emperor title, use `yesman`. Now you're able to offer vassalization to everyone around you and they will agree. Thus you will get realm size and developed province for your future capital.

Also, use `prestige 10000` to get 10000 prestige.

The Gorian Empire should work like the vanilla Roman Empire. Check everything important is included and working.
